python-lib/sharepoint_client.py

```diff
@@ -447,15 +469,17 @@ def __init__(self, sharepoint_user_name, sharepoint_password, sharepoint_tenant,
 
     def get(self, url, headers=None, params=None):
         headers = {} if headers is None else headers
-        headers["accept"] = DSSConstants.APPLICATION_JSON
+        headers["Accept"] = DSSConstants.APPLICATION_JSON
         headers["Authorization"] = self.get_authorization_bearer()
         return requests.get(url, headers=headers, params=params)
 
-    def post(self, url, headers=None, json=None, data=None):
-        headers = {} if headers is None else headers
-        headers["accept"] = DSSConstants.APPLICATION_JSON
-        headers["Authorization"] = self.get_authorization_bearer()
-        return requests.post(url, headers=headers, json=json, data=data)
+    def post(self, url, headers={}, json=None, data=None):
```

Comment:
It is always better to avoid using mutable objects for default arguments as it can lead to errors very painful to debug.